crates/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -37,6 +36,11 @@ pub enum StringFunction {
         ignore_nulls: bool,
     },
     #[cfg(feature = "regex")]
+    ContainsRegex {
+        #[cfg_attr(feature = "serde", serde(skip))]
+        regex: Option<RegexWrap<Arc<Regex>>>,
```

Comment:
Is there a case where this is None?

Reply:
Not directly, but we need it for default implementation